src/app/core/components/layout/layout.component.ts

```diff
@@ -1,15 +1,65 @@
 import { Component, OnInit } from '@angular/core';
+import {AuthenticationService} from "../../services/authentication.service";
+import {Router} from "@angular/router";
+import {Roles} from "../../enums/Roles";
 
 @Component({
   selector: 'app-layout',
   templateUrl: './layout.component.html',
   styleUrls: ['./layout.component.css']
 })
 export class LayoutComponent implements OnInit {
-
-  constructor() { }
+  userRole = Roles[0];
+  btoRole = Roles[1];
```

Comment:
I guess will be better to read roles from enum, in case these roles will be changed

Reply:
Done.